app/src/main/AndroidManifest.xml

```diff
@@ -30,7 +30,7 @@
 
 		<meta-data
 			android:name="com.google.android.geo.API_KEY"
-			android:value="www.google.com" />
+			android:value="@string/GOOGLE_MAPS_API_KEY"/>
```

Comment:
should this be capitalized? @ningning621 @AAAstorga

Reply:
@connorreinhold is this value being pulled from a `strings.xml` file or the gradle?

Reply:
Regarding the capitalization, I can't find an example where an id is all uppercase, so it might be best to switch it back to lowercase.